crates/revm/src/processor.rs

```diff
@@ -50,8 +50,6 @@ use tracing::{debug, trace};
 ///
 /// InspectorStack are used for optional inspecting execution. And it contains
 /// various duration of parts of execution.
-// TODO: https://github.com/bluealloy/revm/pull/745
-// #[derive(Debug)]
```

Comment:
no derive debug necessary anymore?

Reply:
Evm doesn't implement Debug, and doesn't make much sense for this type tbh